api/src/main/resources/liquibase.xml

```diff
@@ -56,7 +56,7 @@
 		<addForeignKeyConstraint baseTableName="reporting_report_design" baseColumnNames="retired_by" constraintName="retired_by_for_reporting_report_design" referencedTableName="users" referencedColumnNames="user_id"	deferrable="false" initiallyDeferred="false" />
 	</changeSet>
 
-	<changeSet id="reporting_report_design_2" author="mseaton">
+	<changeSet id="reporting_report_design_2" author="mseaton" dbms="mysql">
```

Comment:
Did you confirm that this does not affect the checksum?

Reply:
@dkayiwa , yes and also checked with the docs [here](https://www.liquibase.org/blog/what-affects-changeset-checksums) .